### ISSUE TITLE
failing cookie session authentication

### ISSUE THREAD

**Issue by teaforthecat** [teaforthecat/bones.http#2]
see the failing test here #1

**Comment by teaforthecat** [comment]
Present work-around is to use the token instead of the session.  
1. store the token from the login response in a cookie
2. use the cookie's value as the "Authorization" header (with "Token " prepended to it)

**Comment by teaforthecat** [comment]
I've confirmed the cookie session can authenticate with this:

``` bash
curl "localhost:3000/api/events" --cookie "bones-session=w0Sr6aFcdwmtcnbmuxfmiuYgtmo0VUGylXKtdpVpv%2B7c5y3FVjWYdJ21cgv2g%2BlNkgdyYvE%2BvpqQWHUW%2BhcobzDOqd8M6SIWVWtp%2FXP9wG4%3D--fTYQgX1%2BSQkz2Q%2Bc5qaEhYGPMlBU2hdcdSy5bcs%2B2S0%3D" -v
```

**Comment by teaforthecat** [comment]
Ok the problem was the cookie parsing it the test itself. Now that that is fixed and passing, I believe the next place to look is the CORS interactions. I've seen the session fail to authenticate when the browser is on a different port than the server (for cljs development for example). Perhaps the cookie isn't reaching the server for some reason.

**Comment by teaforthecat** [comment]
There was an issue with the encoding of the cookie, which is fixed in the refactoring of 0d761b1
